be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -688,8 +688,9 @@ Status VNodeChannel::close_wait(RuntimeState* state) {
     }
 
     // waiting for finished, it may take a long time, so we couldn't set a timeout
-    while (!_add_batches_finished && !_cancelled) {
-        std::this_thread::sleep_for(std::chrono::milliseconds(1));
+    while (!_add_batches_finished && !_cancelled && !state->is_cancelled()) {
+        // std::this_thread::sleep_for(std::chrono::milliseconds(1));
+        bthread_usleep(1000);
```

Comment:
Please not use bhtread here, @chenlinzhong will add pthread pool after internal_service.cpp for every brpc request

Reply:
Here, there is another bug, if a fragment is not open, then close_wait does not handle cancell. bthread_sleep works like sleep in pthread. so I merged it.